src/components/site-footer/site-footer.html.twig

```diff
@@ -141,6 +142,9 @@
           {% if _sections.links_inline %}
             {% set _list_class_name = _list_class_name ~ ' ecl-site-footer__list--inline' %}
           {% endif %}
+          {% if _sections.links_separator %}
```

Comment:
brilliant, that's all we needed..? :)

Reply:
for the EC core footer yes. We'll see later for the other ones...
And in any case I still think that it is worth refactoring this component